packages/design-system-kit/src/components/LanguageSwitcher/LanguageSwitcher.jsx

```diff
@@ -0,0 +1,93 @@
+    };
+
+    const uppercaseFirst = string => {
+        return string.charAt(0).toUpperCase() + string.substr(1);
+    };
+
+    return (
+        <React.Fragment>
+            {languages.length > 1 ? (
+                <React.Fragment>
+                    <Button aria-owns={anchorEl ? 'language-switcher' : null}
+                            aria-haspopup="true"
+                            data-cm-role="language-switcher"
+                            size="compact"
+                            color={color}
+                            onClick={handleClick}
+                    >
+                        <Typography noWrap variant="zeta" color="inherit">
+                            {uppercaseFirst(_.find(languages, language => language.language === lang).displayName)}
+                            &nbsp;
+                        </Typography>
+                        <ChevronDown fontSize="small" color="inherit"/>
+                    </Button>
+                    <Menu id="language-switcher"
+                          anchorEl={anchorEl}
+                          open={Boolean(anchorEl)}
+                          onClose={handleClose}
+                    >
+                        {languages.map(lang => {
+                            return (
+                                <MenuItem
+                                    key={lang.language}
+                                    onClick={() => {
+                                        onSelectLanguage(lang.language);
+                                        handleClose();
+                                    }}
+                                >
+                                    {uppercaseFirst(lang.displayName)}
+                                </MenuItem>
+                            );
+                        })}
+                    </Menu>
+                </React.Fragment>
+            ) : (
+                <React.Fragment>
+                    <Button aria-owns={anchorEl ? 'language-switcher' : null}
+                            aria-haspopup="true"
+                            data-cm-role="language-switcher"
+                            size="compact"
+                            color={color}
+                            onClick={handleClick}
+                    >
+                        <Typography noWrap variant="zeta" color="inherit">
+                            {uppercaseFirst(_.find(languages, language => language.language === lang).displayName)}
+                            &nbsp;
+                        </Typography>
+                    </Button>
+                </React.Fragment>
+            )}
+        </React.Fragment>
+    );
+};
+
+LanguageSwitcher.propTypes = {
+    color: PropTypes.string,
+    lang: PropTypes.string.isRequired,
+    languages: PropTypes.arrayOf(PropTypes.object).isRequired,
+    onSelectLanguage: PropTypes.func.isRequired
+};
+
+LanguageSwitcher.defaultProps = {
+    color: 'default'
+};
+
+export default LanguageSwitcher;
```

Comment:
Minor: You can add displayName for component debugging.
>LanguageSwitcher.displayName = 'LanguageSwitcher';

Reply:
done